src/test/kotlin/com/igorwojda/integer/generateallpairs/challenge.kt

```diff
@@ -4,7 +4,13 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getAllPairs(n: Int): List<Pair<Int, Int>> {
-    TODO("not implemented")
+    var list = arrayListOf<Pair<Int,Int>>()
+    for(i in 0..n){
+        for(j in 0..n){
+            list.add(Pair(i,j))
```

Comment:
Can you think of an infix function (https://kotlinlang.org/docs/reference/functions.html#infix-notation) that can make your code a lil bit more readable?